new-frontiers-14/frontier-station-14
# Move Frontier Changelog to a separate file.

## About the PR



Moved Frontier changelogs to a separate folder, adjusted fluent entries for changelog tabs, pointed changelog scripts to new Frontier.yml changelog, trimmed Frontier changelog to 500 entries using manual_changelog.py.

## Why / Balance


Pulling in upstream changes results in a big changelog conflict.  This should fix that, and GitHub changes should go into the proper changelog.

## How to test


1. Run manual_changelog.py, type a few entries, hit ctrl+c, you should see entries in your proper file and in-game.
2. Check the output file for each of the scripts (should point to Resources/Changelog/Frontier.yml).
3. Open the game, check the Changelog, should be in the correct order.

## Media


![image](https://github.com/user-attachments/assets/19863993-9bf3-4eab-9aa3-f7d50260828f)

## Requirements


## Breaking changes


**Changelog**



:cl:
- tweak: Frontier changelog now shows up under the Frontier tab.
